plugin/resources/com/microsoft/alm/plugin/idea/ui/tfplugin.properties

```diff
@@ -406,7 +406,7 @@ Tfvc.Checkin.Successful.Msg=Successfully created {0}
 Tfvc.Checkin.Link.Text=Changeset #{0}
 Tfvc.Checkin.Status=Checking in files...
 Tfvc.Notification.FileNameStartsWithDollar=File name starts with '$', TFVC client will not work properly until this file is ignored\: {0}
-Tfvc.RepositoryImportError=Couldn't create TFVC repository in {0}
+Tfvc.RepositoryImportError=Couldn''t create TFVC repository in {0}
```

Comment:
'' is it escaping?

Reply:
Yup. See the `MessageFormat` [documentation](https://docs.oracle.com/javase/7/docs/api/java/text/MessageFormat.html):

> A single quote itself must be represented by doubled single quotes `''` throughout a _String_.

I was surprised, too :)